src/compiler/utilities.ts

```diff
@@ -1177,6 +1177,40 @@ namespace ts {
         return false;
     }
 
+    // Return true if the given identifier is classified as an IdentifierName

```

Comment:
An IdentifierName is any identifier, including reserved words. It's part of the ES grammar and we use the term in many places in the compiler.